backends/xnnpack/_passes/channels_last_tagged_reshape_pass.py

```diff
@@ -333,7 +344,10 @@ def input_to_nchw(
             # do anything else here
             self.mark_as_nchw_node(input_node)
 
-        if self.is_nchw_node(input_node):
+        if input_node.op == "placeholder":
```

Comment:
do we need to check that this is a placeholder? Can we just skip if the input is already contiguous?